auto/unix

```diff
@@ -311,6 +311,16 @@ ngx_feature_test="cpu_set_t mask;
                   sched_setaffinity(0, sizeof(cpu_set_t), &mask)"
 . auto/feature
 
+ngx_feature="sched_getaffinity()"
+ngx_feature_name="NGX_HAVE_SCHED_GETAFFINITY"
```

Comment:
if this macro "NGX_HAVE_SCHED_GETAFFINITY" does not come from the official nginx, you could rename it to "T_NGX_HAVE_SCHED_GETAFFINITY". Adding the prefix "T_" could distinguish if this feature comes from Tengine.